tensorflow/lite/kernels/reduce.cc

```diff
@@ -618,7 +618,6 @@ TfLiteStatus EvalType<bool>(TfLiteContext* context, TfLiteNode* node,
                              [](const bool current, const bool in) -> bool {
                                return in || current;
                              });
-      break;
```

Comment:
Why it doesn't remove the break in kAll case as well?

Reply:
Presumably the template was only instantiated with kAny, so that's the only case I was warned about.  Fixed.